pkg/apiserver/registry/networkpolicy/clustergroupmember/rest_test.go

```diff
@@ -30,11 +31,17 @@ type fakeQuerier struct {
 	members map[string]controlplane.GroupMemberSet
 }
 
-func (q fakeQuerier) GetGroupMembers(uid string) (controlplane.GroupMemberSet, error) {
+func (q fakeQuerier) GetGroupMembers(uid string) (controlplane.GroupMemberSet, []controlplane.IPBlock, error) {
 	if memberList, ok := q.members[uid]; ok {
-		return memberList, nil
+		return memberList, nil, nil
+	} else if uid == "cgIPBlock" {
+		testCIDR := controlplane.IPNet{
+			IP:           controlplane.IPAddress(net.ParseIP("10.0.0.1")),
+			PrefixLength: int32(24),
+		}
+		return nil, []controlplane.IPBlock{{CIDR: testCIDR}}, nil
 	}
-	return controlplane.GroupMemberSet{}, nil
+	return nil, nil, nil
```

Comment:
it's not a great way to write that test, hardcoding the uid used in a test case below like this

it feels like `fakeQuerier` could have 2 maps instead of 1: one map for `GroupMemberSet`s and one map for `IPBlock`s.

Reply:
oops i missed this one

Reply:
done